src/daft-proto/proto/v1/daft.proto

```diff
@@ -201,30 +201,25 @@ message Literal {
     bool boolean = 2;
     string utf8 = 3;
     bytes binary = 4;
-    FixedSizeBinary fixed_size_binary = 5;
-    int32 int8 = 6;
-    uint32 uint8 = 7;
-    int32 int16 = 8;
-    uint32 uint16 = 9;
-    int32 int32 = 10;
-    uint32 uint32 = 11;
-    int64 int64 = 12;
-    uint64 uint64 = 13;
-    Timestamp timestamp = 14;
-    int32 date = 15;
-    Time time = 16;
-    Duration duration = 17;
-    Interval interval = 18;
```

Comment:
I am aware generally for protos we do not want to change the numbers but instead add them to a `reserved` section if we are removing a variant. However since the protos are still in development I am hoping this is okay. Let me know if we should do something else here though.